.circleci/publish.sh

```diff
@@ -19,7 +19,7 @@ echo "  <servers>" >> ~/.m2/settings.xml
 echo "    <server>" >> ~/.m2/settings.xml
 echo "      <id>ossrh</id>" >> ~/.m2/settings.xml
 echo "      <username>$OSSRH_USERNAME</username>" >> ~/.m2/settings.xml
-echo "      <password>$OSSRH_PASSWORD</password>" >> ~/.m2/settings.xml
+echo "      <password><![CDATA[$OSSRH_PASSWORD]]></password>" >> ~/.m2/settings.xml
```

Comment:
What does this do?

Reply:
Our OSSRH password contains an ampersand (`&`), which in XML is a special character. It happens in this case to make the XML parser think we want to reference an XML entity: "entity reference names can not start with character '*'".

CDATA tells XML that the inside is just text, not formatting. So putting the password in CDATA tells XML that the ampersand is just an ampersand.